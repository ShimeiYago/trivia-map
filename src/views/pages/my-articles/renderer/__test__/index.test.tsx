import { shallow, ShallowWrapper } from 'enzyme';
import { Renderer, Props } from '..';

let wrapper: ShallowWrapper<Props, null, Renderer>;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<Renderer />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('with user', () => {
    wrapper.setProps({
      user: {
        userId: 1,
        email: 'xxx@example.com',
        nickname: 'xxx',
        icon: 'https://xxx.com/xxx.png',
        isSocialAccount: false,
      },
    });
    expect(wrapper).toMatchSnapshot();
  });
});
