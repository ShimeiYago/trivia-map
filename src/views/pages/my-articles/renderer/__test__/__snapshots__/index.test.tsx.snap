// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<AdminWrapper>
  <ForwardRef(Stack)
    spacing={3}
  >
    <BackToNavi
      link="/admin"
      text="マイページ"
    />
    <ForwardRef(Box)>
      <ForwardRef(Typography)
        align="center"
        component="h2"
        sx={
          Object {
            "mb": 3,
          }
        }
        variant="h4"
      >
        投稿一覧
      </ForwardRef(Typography)>
      <ArticleManageTable />
    </ForwardRef(Box)>
  </ForwardRef(Stack)>
</AdminWrapper>
`;
